doc/command-line.md

```diff
@@ -993,6 +993,39 @@ size: 5177
 signature license: CC0
 ```
 
+### `sourmash signature fileinfo` - display a summary of the contents of a sourmash collection
+
+Display signature file, database, or collection.
+
+For example,
+```
+sourmash sig fileinfo tests/test-data/prot/all.zip
+```
+will display:
+```
+path filetype: ZipFileLinearIndex
+location: /Users/t/dev/sourmash/tests/test-data/prot/all.zip
+is database? yes
+has manifest? yes
+is nonempty? yes
+num signatures: 8
+** examining manifest...
```

Comment:
do manifests have an associated creation date/datetime that we could print? Perhaps this is more related to manifest metadata (#1847)?

Reply:
they do not, alas.